electionleaflets/apps/leaflets/tests/test_models.py

```diff
@@ -23,14 +24,22 @@ def test_markdown_error(self):
         leaflet = Leaflet.objects.create(title="Test Leaflet", description=None)
         response = self.client.get(leaflet.get_absolute_url())
         self.assertEqual(response.status_code, 200)
+        
+    def test_redirect_to_leaflet(self):
```

Comment:
I don't think this is testing a redirect? It's asserting that a leaflet URL exists. Can you rename this test?

Reply:
Renamed with https://github.com/DemocracyClub/electionleaflets/pull/257/commits/55f6a1f8661a879b1512904d0c8c294bf7043dc0